djmoney/money.py

```diff
@@ -90,15 +89,10 @@ def get_current_locale():
 def convert_money(value, currency):
     """
     Converts other Money instances to the local currency.
-    If django-money-rates is installed we can automatically perform operations with different currencies.
     """
     if getattr(settings, 'AUTO_CONVERT_MONEY', False):
-        if 'djmoney_rates' in settings.INSTALLED_APPS:
-            try:
-                from djmoney_rates.utils import convert_money
-
-                return convert_money(value.amount, value.currency, currency)
-            except ImportError:
-                raise ImproperlyConfigured('djmoney_rates supports only Django 1.8')
-        raise ImproperlyConfigured('You must install djmoney-rates to use AUTO_CONVERT_MONEY = True')
+        from djmoney.contrib.exchange.models import get_rate
+
+        amount = value.amount * get_rate(value.currency, currency)
+        return Money(amount, currency)
```

Comment:
It seems like, this function was designed for direct usage in ``Money`` and it works as no-op if ``AUTO_CONVERT_MONEY`` is not set.

I agree that it is strange if you want to directly convert money and it is no-op :) It should fail (for example, with ``MissingRate`` exception) if there is something wrong with setup or with rates data. Probably we could have something like this:

```python
# money.py
from djmoney.contrib.exchange.models import convert_money


def maybe_convert(value, currency):
    """
    For usage in ``Money``.
    """
    if getattr(settings, 'AUTO_CONVERT_MONEY', False):
        return convert_money(value, currency)
    return value


# djmoney.contrib.exchange.models
def convert_money(value, currency):
    if 'djmoney.contrib.exchange' not in settings.INSTALLED_APPS:
        raise ImproperlyConfigured('You have to add \'djmoney.contrib.exchange\' to your project\'s INSTALLED_APPS in order to use currency exchange.')
    amount = value.amount * get_rate(value.currency, currency)
    return value.__class__(amount, currency)
```

In this case, the current workflow with ``Money`` instances will remain the same, but the direct usage of ``convert_money`` will be more straightforward.
What do you think?

Reply:
That is way more explicit :+1: 

So the use of `maybe_convert` is that someone wants to  *maybe* store a price in a converted currency, but if there is no conversion, it would just keep the existing currency. This seems understandable, in case some reusable cannot predict its environment.

Reply:
Initially, I thought, that it could be a private function to use **only** in ``Money.__sub__`` and ``Money.__add__`` to preserve current behavior, but I'm not sure if it could be useful as a public function.